lib/Serialization/Deserialization.cpp

```diff
@@ -3743,7 +3743,7 @@ class swift::DeclDeserializer {
       return declOrOffset;
 
     auto extension = ExtensionDecl::create(ctx, SourceLoc(), nullptr, { },
-                                           DC, nullptr);
+                                           nullptr, DC, nullptr);
```

Comment:
Is the plan to handle serialization and deserialization in a later PR?

Reply:
I think that can be handled in this PR, I just need to figure out everything else right now and then handle that.